indra/sources/omnipath/processor.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import unicode_literals
+import logging
+from collections import Counter
+from indra.preassembler.grounding_mapper import GroundingMapper
```

Comment:
It would be better to use `from indra.ontology.standardize import standardize_agent_name` and use that below.